src/st/class-wpml-pb-string-registration.php

```diff
@@ -65,16 +65,27 @@ public function get_string_title( $string_id ) {
 	}
 
 	/**
-	 * @param int $post_id
-	 * @param string $content
-	 * @param string $type
-	 * @param string $title
-	 * @param string $name
-	 * @param int $location
+	 * Register string.
+	 *
+	 * @param int    $post_id  Post Id.
+	 * @param string $content  String content.
+	 * @param string $type     String editor type.
+	 * @param string $title    String title.
+	 * @param string $name     String name.
+	 * @param int    $location String location.
+	 * @param string $wrap_tag String wrap tag.
```

Comment:
@kagg-design do you see the Scrutinizer notice below?

It's suggesting to change `@return int $string_id` into `@return null|int $string_id` because I guess `$this->get_string_id_from_package` may return a null value.

Reply:
@andreasciamanna done.